NBitcoin/BIP32/ExtPubKey.cs

```diff
@@ -72,12 +72,23 @@ internal ExtPubKey()
 		{
 		}
 
+        	/// <summary>

```

Comment:
Unneccessary whitespace.

Reply:
https://github.com/MetacoSA/NBitcoin/pull/477

Reply:
It's really weird, the formatting seems to be inconsistent on Linux (I didn't see the whitespace in my editor), and the same with the indentation problem I fixed above. Good catch!